tools/setup-protobuf.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Running OTLP exporter on old distros requires new `protoc3`, i.e. I'm fixing here something that never worked before on Ubuntu 14.04. Now it works great.